.editorconfig

```diff
@@ -0,0 +1,5 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
.editorconfig is not related to the accuracy based job attributes. Please, remove it from this pull request.